inc/common/admin-bar.php

```diff
@@ -22,7 +22,7 @@ function rocket_admin_bar( $wp_admin_bar ) {
 		 *
 		 * @param string $uri Current uri
 		 */
-		$referer = (string) apply_filters( 'rocket_admin_bar_referer', esc_url( $uri ) );
+		$referer = (string) apply_filters( 'rocket_admin_bar_referer', esc_url_raw( $uri ) );
```

Comment:
We should escape after the return value of the filter

Reply:
I've changed it :)